### PR TITLE
Add support for provisioning on a specific cluster

### PR DESCRIPTION
**Pull request by tecracer-theinen**
### Description

Implement selection of a cluster to provision to.

### Issues Resolved

Resolves feature request #9.

### Check List

- [X] Functionality has been documented in the README if applicable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,8 +126,10 @@ The following optional parameters should be used in the `driver_config` for the
  - `folder` - Folder into which the new machine should be stored. If specified the folder _must_ already exist.
  - `poweron` - Power on the new virtual machine. Default: true
  - `vm_name` - Specify name of virtual machine. Default: `<suite>-<platform>-<random-hexid>`
- - `resource_pool` - Name of the resource pool to use when creating the machine. Will search first pool by default, can use value 'Resources' for none.
- - `clone_type` - Type of clone, will default to "full" to create complete copies of template. Needs a VM as template parameter, if "linked" clone desired.
+
+Only one of the following optional parameters can be given:
+ - `resource_pool` - Name of the resource pool to use when creating the machine. Will use first pool by default.
```
Comment (kagarmoe) [inline README.md]:
```suggestion
 - `resource_pool` - Name of the resource pool to use when creating the machine. Default: first pool
```

File: README.md [hunk 2]
```diff
@@ -126,8 +126,10 @@ The following optional parameters should be used in the `driver_config` for the
...
+ - `cluster` - Cluster on which the new virtual machine should be created. Will provision in the root pool, if paramter given and choose a targethost within
```
Comment (kagarmoe) [inline README.md]:
```suggestion
 - `cluster` - Cluster on which the new virtual machine should be created. Default: Provisions in the root pool. If a parameter is given,  if parameter given, then chooses a `targethost` within.
```
I'm not sure I understand what `cluster` does, particularly when passed with a parameter.

Comment (tecracer-theinen) [reply README.md]: Yeah, that description was definitively too fuzzy. 

What about "Default: picks first cluster found in vCenter"

**Comment by tas50** [comment]
@tecracer-theinen Any chance you can rebase this?

**Pull request merged**